x-pack/plugins/actions/server/actions_client.test.ts

```diff
@@ -1703,6 +1707,11 @@ describe('update()', () => {
 
       expect(authorization.ensureAuthorized).toHaveBeenCalledWith('update');
     });
+
+    test(`deletes any existing authorization tokens`, async () => {
+      await actionsClient.delete({ id: '1' });
```

Comment:
Should this be `await updateOperation()` to test that the connector gets deleted on update?

Reply:
Doh! Yup, copy paste issue. I fixed this in this commit: https://github.com/elastic/kibana/pull/132061/commits/f148ffa329681f68afe87bdbd05d274f683946cf.